R/slide.R

```diff
@@ -208,129 +206,198 @@ epi_slide <- function(
     }
     return(f_wrapper)
   }
+  epi_slide_one_group_partial <- function(.data_group, .group_key, ...) {
+    epi_slide_one_group(
+      .data_group, .group_key, ...,
+      f_factory = f_wrapper_factory,
+      before = window_args$before,
+      after = window_args$after,
+      ref_time_values = .ref_time_values,
+      all_rows = .all_rows,
+      new_col_name = .new_col_name,
+      used_data_masking = used_data_masking,
+      time_type = time_type,
+      complete_only = .complete_only
+    )
+  }
```

Comment:
just a minor personal preference for making my own partial functions rather than letting some downstream code handle it. minor distrust of other code xD.

i prefer to keep epi_slide_one_group outside - one thing i don't like about keeping a giant work function inside another function is needing to scroll down through the whole body of `epi_slide_one_group` to see what the end of `epi_slide` is doing. i also just find it more natural to read: the broad structure of epi_slide as (1) validation, (2) group_modify with group-specific code, (3) stitching together is much clearer to me this way.

Reply:
Hm I see your point, so I inlined it as a lambda again, but kept the variables in. I mostly just disliked using `group_modify` to send arguments to the lambda `.f` because with `.keep = FALSE` there, we're mixing arguments the lambda and group_modify itself. Let me know if that feels better.